### PR TITLE
at cron file now lives in /etc/cron.d/at

### PR DESCRIPTION
**Pull request by kravietz**
Fix https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=243380

### CODE REVIEW

File: usr.sbin/cron/crontab/crontab.5 [hunk 1]
```diff
@@ -151,11 +151,15 @@ usually does not read its mail.
 .Pp
 The format of a cron command is very much the V7 standard, with a number of
 upward-compatible extensions.
-Each line has five time and date fields,
-followed by a user name
-(with optional ``:<group>'' and ``/<login-class>'' suffixes)
-if this is the system crontab file,
+
```
Comment (igalic) [inline usr.sbin/cron/crontab/crontab.5:154]: empty lines is not how you generate a paragraph in mandoc, you use `.Pp`

File: usr.sbin/cron/crontab/crontab.5 [hunk 2]
```diff
@@ -151,11 +151,15 @@ usually does not read its mail.
...
+Each user cron line has five time and date fields, followed by a command.
+
+Each line in system crontab (
+.Pa /etc/crontab, /etc/cron.d, /usr/local/etc/cron.d
+) has five time and date fields, followed by a valid user name
+(with optional ``:<group>'' and ``/<login-class>'' suffixes),
 followed by a command.
+
```
Comment (igalic) [inline usr.sbin/cron/crontab/crontab.5:162]: use `.Pp` if you want to generate a new paragraph.

File: usr.sbin/cron/crontab/crontab.5
Comment (igalic) [inline usr.sbin/cron/crontab/crontab.5:1]: If you think this is a significant change, then this file needs a `.Dd` bump

**Review by igalic — CHANGES_REQUESTED** [review]
This doesn't need to be 3 commits.

also see inline comments.

(I still wish github, or literally any forge out there, would render nroff / mdoc files)

**Comment by kravietz** [comment]
@igalic Thanks for all suggestion, fixed! Commits can be squashed into one on merge, if I remember correctly.

**Comment by emaste** [comment]
> Commits can be squashed into one on merge, if I remember correctly.

We don't (can't) use the GitHub merge functionality, as GitHub is not the primary repository and does not have direct write access. It is possible for the FreeBSD committer who lands this to edit or squash the commits but we prefer that pull requests have commits prepared in a ready-to-land state.

Also please note from CONTRIBUTING.md:
* Commits should include one or more `Signed-off-by:` lines with full name and email address certifying [Developer Certificate of Origin](https://developercertificate.org/).

File: usr.sbin/cron/crontab/crontab.5 [hunk 3]
```diff
@@ -281,7 +285,16 @@ Note, however, that overlap may occur if the job is running when the file
 containing the job is modified and subsequently reloaded.
 The first run is scheduled for the specified number of seconds after cron
 is started or the crontab entry is reloaded.
-.Sh EXAMPLE CRON FILE
+
```
Comment (bsdimp) [inline usr.sbin/cron/crontab/crontab.5:288]: This is an extra newline and should be removed.

**Comment by bsdimp** [comment]
Pushed with minor tweaks.

**Comment by bsdimp** [comment]
Sorry this took so long to land... Thanks for fixing this and cross referencing the bug.

**Pull request merged**